### PR TITLE
Individual Sinks Documentation Updated

### PR DESCRIPTION
**Pull request by kranurag7**
- updated the documentation of Individual sinks including datadog, kafka, msteams, opsgenie and webhook. 
- In order to avoid same content on two pages, we still need to remove the content from index.rst

### CODE REVIEW

File: docs/catalog/sinks/DataDog.rst [hunk 1]
```diff
@@ -0,0 +1,25 @@
+DataDog 
+##########
+
+Robusta can send playbooks results to the Datadog events API.
+
+To configure datadog sink, we need DataDog API key. The API key can be retrieved from your DataDog Account.
```
Comment (aantn) [inline docs/catalog/sinks/DataDog.rst:6]: Should be `need a DataDog`

File: docs/catalog/sinks/DataDog.rst [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+.. admonition:: values.yaml
+
+    .. code-block:: yaml
+
+        sinks_config:
+        - datadog_sink:
+            name: datadog_sink
+            api_key: "datadog api key"
+            default: false
```
Comment (aantn) [inline docs/catalog/sinks/DataDog.rst:19]: I would remove the `default: false` from here and the other sink examples.

File: docs/catalog/sinks/Opsgenie.rst [hunk 3]
```diff
@@ -0,0 +1,31 @@
+Opsgenie
+##########
+
+Robusta can send playbooks results to the OpsGenie alerts API.
+
+To configure OpsGenie, We need OpsGenie API key. It can be configured using OpsGenie team integration.
```
Comment (aantn) [inline docs/catalog/sinks/Opsgenie.rst:6]: Should be `need an OpsGenie`

Comment (aantn) [reply docs/catalog/sinks/Opsgenie.rst:6]: Should be `using the OpsGenie team integration`

File: docs/catalog/sinks/DataDog.rst [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+
+**Datadog:**
```
Comment (aantn) [inline docs/catalog/sinks/DataDog.rst:21]: I think we can remote the `**Datadog**` here and just add a heading called `Screenshots` or `Example Output`. Same applies to the other sinks.

File: docs/catalog/sinks/kafka.rst [hunk 5]
```diff
@@ -0,0 +1,25 @@
+Kafka 
+######### 
+
+Robusta can send playbook result to a Kafka topic.
```
Comment (aantn) [inline docs/catalog/sinks/kafka.rst:4]: Should be `results`

File: docs/catalog/sinks/ms-teams.rst [hunk 6]
```diff
@@ -0,0 +1,24 @@
+MS teams
+##########
+
+Robusta can send playbook result to MS teams channel webhook.
```
Comment (aantn) [inline docs/catalog/sinks/ms-teams.rst:4]: Should be `results to a MS teams...`

File: docs/catalog/sinks/ms-teams.rst [hunk 7]
```diff
@@ -0,0 +1,24 @@
...
+
+To configure MS teams sink we need a webhook URL for the target teams channel. You can configure it in MS teams channel connectors.
```
Comment (aantn) [inline docs/catalog/sinks/ms-teams.rst:6]: Should be `a MS teams sink`

**Review by aantn — COMMENTED** [review]
Very nice. Added some comments re. typos and other minor stuff.

As soon as index.rst is updated, lets get this merged.

**Comment by aantn** [comment]
Looks good. Can you update `catalog/sinks/index.rst` as well? We can remove the duplicate information from there. Don't forget to also update the `toctree` at the bottom of the file - that is what generates the links to the pages in the sidebar.

**Comment by kranurag7** [comment]
I will remove the duplicate text from the index.rst page.

**Comment by kranurag7** [comment]
@aantn I have updated the `index.rst` page. Also I have given the links to html pages of the particular sink.

**Comment by kranurag7** [comment]
@aantn I have made the changes. Have a look and I think if the `ref` is case-sensitive then it's going to create problem in rendering. If it is so, then I will rename the file in the `Title` format. 

>

**Comment by aantn** [comment]
@kranurag7 looks great, merging! 

I tested this and the refs all work. Note that you can always test the rendering yourself by running `./docs_autobuild.sh`.

**Comment by kranurag7** [comment]
Thank you @aantn I will take care of that in future.

**Pull request merged**